src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -1056,10 +1060,44 @@ public void testExecuteAndPostProcessRubricQuestion() {
         assertEquals(Const.FeedbackQuestion.RUBRIC_ERROR_INVALID_WEIGHT, result.getStatusMessage());
 
         expectedLogMessage = "TEAMMATESLOG|||instructorFeedbackQuestionAdd|||"
-                                    + "instructorFeedbackQuestionAdd|||true|||"
-                                    + "Instructor|||Instructor 1 of Course 1|||"
-                                    + "idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
-                                    + "Unknown|||/page/instructorFeedbackQuestionAdd";
+                + "instructorFeedbackQuestionAdd|||true|||"
+                + "Instructor|||Instructor 1 of Course 1|||"
+                + "idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
+                + "Unknown|||/page/instructorFeedbackQuestionAdd";
+        AssertHelper.assertLogMessageEquals(expectedLogMessage, action.getLogMessage());
+
+        ______TS("Failure case: Negative weights attached");
```

Comment:
Oops! I missed this test. My apologies. 

Actually I mistakenly typed the paramName as 
`Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT + "1-1", "3"`, instead of
`Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT + "-1-1", "3"` :man_facepalming: Which is why the weights of this test were not being parsed at all resulting in invalid size error which in turn became a reason for this test to pass!